assets/src/edit-story/components/canvas/useInsertElement.js

```diff
@@ -83,6 +83,11 @@ function useInsertElement() {
           }
         }, 0);
       }
+      // When done, send fake focusout event to document to force focus to new element
+      setTimeout(() => {
+        const evt = new window.FocusEvent('focusout');
+        window.document.dispatchEvent(evt);
+      }, 1);
```

Comment:
Why not just `0`?

Reply:
To be after the above `setTimeout`? Okay, I'll remove the argument (0 is default).